Added ski and dosl include and redo the team and smv include

maceq687 opened the pull request:
From Niels:

> We are now SKI-suppliers. Can we fit the logo in somewhere? Maybe near the smv:digital section?

And:

> Hmm we could def add [dosl.dk](http://dosl.dk/) as well

To see changes visit the homepage https://deploy-preview-159--zealous-fermat-77bc7b.netlify.app/ and scroll down to the sections that appear after the centered button with label "See examples of what we have done"

maceq687 committed:
Added ski and dosl include

maceq687 committed:
Restructured team and smv digital into sepparate certification and team sections

maceq687 committed:
Improved who we are section styling

maceq687 committed:
Removed duplicated style and made button width auto by default

maceq687 committed:
Added padding to team section images on wide screens

maceq687 committed:
Adjusted team section images blending

maceq687 committed:
Replaced SKI logo

maceq687 committed:
Applied deranged transforms for team section images

maceq687 committed:
Removed titles from content in certification section

maceq687 committed:
Added box shadow to team section images

maceq687 committed:
Added text to certified section and adjusted padding for certified and team sections

maceq687 committed:
Skewed certification section and adjusted logo sizing

maceq687 committed:
Updated smv digital logo, adjusted certification section logo sizing

maceq687 committed:
Make certification section with sqaure logo follow horizontal layout

maceq687 committed:
Adjusted padding of the certified sec tions on small screens and limited logo max height

maceq687 committed:
Scaled all certified section logos on small screens

maceq687 committed:
Added extra top margin to certification section element with square logo

maceq687 committed:
Team section members photos added transform on scroll in and out of the view

lasseborly commented on _includes/certification.html:
```diff
@@ -0,0 +1,24 @@
+      <img src="/img/smv_digital.svg" alt="SMV:Digital" class="certification-section__content-logo" loading="lazy" />
+      <p>
+        As registered advisors at
+        <a href="https://smvdigital.dk/content/ydelser/tilbud/4e84e652-3d63-4b24-b5f5-a278d2ad27e6/" target="_blank" rel="noopener">SMV:Digital</a>
```
Let's not have target="_blank" here.
https://www.w3.org/WAI/WCAG21/Techniques/html/H83.html

lasseborly replied:
The same for the rest of the logo links.

lasseborly replied:
Also the rel="noopener"

maceq687 replied:
Fixed in 7586acd

lasseborly commented on _includes/team.html:
```diff
@@ -0,0 +1,92 @@
+        <div><img src="/img/people/haidar.jpg" alt="" loading="lazy" class="team_member"/></div>
+        <div><img src="/img/people/philip-dein.jpg" alt="" loading="lazy" class="team_member"/></div>
+        <div><img src="/img/people/anders.jpg" alt="" loading="lazy" class="team_member"/></div>
+        <div><img src="/img/people/niels-abildgaard.jpg" alt="" loading="lazy" class="team_member"/></div>
```
Empty alt tags?

maceq687 replied:
Assumption here is that the images are here for decoration

lasseborly commented on _sass/button.scss:
```diff
@@ -8,7 +8,7 @@
   display: inline-block;
   border: none;
-  width: 100%;
+  width: auto;
```
This seems like quite a big change. Did we look at how this affects the rest of the website?

maceq687 replied:
I went through the entire website once more and the only place that it changes the button element appear to be in the contact form. This perhaps could be fixed by adding a modifying class to it i.e. button--wide and remove the definition for class button--narrow since it would make sense for the buttons to not have width 100% by default. Would that make sense?

maceq687 replied:
We will do that

maceq687 replied:
Relevant commit 7c3d404

lasseborly commented on css/home.scss:
```diff
@@ -824,61 +849,57 @@ permalink: /css/homestyle.css
       object-fit: cover;
       height: 100%;
       width: 100%;
+      box-shadow: 0 6px 5px rgba(0,0,0,0.2), 0 4px 3px rgba(0,0,0,0.05);
```
Don't we have a variable for these shadows?

maceq687 replied:
We generally use variables sporadically on the website, I don't see any for the shadows

maceq687 replied:
Let's use variables

maceq687 replied:
Relevant commit ec42b785d4d8c34d6ae3223ce7caaa02d8835bf6

lasseborly commented on css/home.scss:
```diff
@@ -789,33 +789,58 @@ permalink: /css/homestyle.css
+    padding-bottom: 6rem;
+  }
+  
+  @media(max-width: 539px) {
```
That is oddly specific? :P

maceq687 replied:
That is one of the breakpoints used around the website for responsivity, those values could also be extracted into variables

maceq687 replied:
Let's use scss varaibles

maceq687 replied:
Relevant commit 3e37d5ae8ec4439442a66c52699018ccd33c2303

lasseborly commented on _includes/certification.html:
```diff
@@ -0,0 +1,24 @@
+      </p>
+    </div>
+    <div>
+      <img src="/img/dosl-logo-white.svg" alt="dosl-logo" class="certification-section__content-logo" loading="lazy" />
```
For the alt text on these logos let's go through the decision W3 tree:

https://www.w3.org/WAI/tutorials/images/decision-tree/

maceq687 replied:
Sure, if we treat them as decorative images it makes sense to keep their alt text empty, changes in 7586acd

lasseborly reviewed (changes requested):
Let's fix these small things together and get this merged!

lasseborly commented:
Also missing danish version?

maceq687 committed:
Certification include - changed links - removed target blank, rel noopener and changed logo images to empty since they serve decorative purpose

maceq687 commented:
> Also missing danish version?

Includes created in this PR are not (yet?) used on the page that has English and Danish variants

maceq687 commented:
Let's make sure when those includes appear on the website that we have appropriate translations

maceq687 committed:
Button: set default width to auto and added modifier class button--wide

maceq687 committed:
Button: added reusable globally style for a button with border and refactored code where it appeared before

maceq687 committed:
Scss: added variables for breakpoints used around the website

maceq687 committed:
Scss: added variable for shadows

maceq687 committed:
Added variables import to homestyle.css

maceq687 committed:
Replaced certification section with team on blog and post layout

Keeping team section on the blog and post layout is possible (to maintain as much content as possible) but making it work with the certification section goes beyond the scope of this PR, especially since its design is skewed and it has a blue background that makes additionally tricky to fit right bellow another blue section (that is the case on both blog page and post layout page)
